person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  def initialize(age, name = 'Unknown', _parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def can_use_services?
+    of_age? || parent_permission ? true : false
+  end
```

Comment:
- [ ] The `parent_permission` here is not an instance variable so it would cause errors. You should use the `@parent_permission` instance. And we don't need to repeat ourselves by checking if it is true or false, since `@parent_permission` is already boolean.

  ```
  def can_use_services?
    of_age? || @parent_permission
  end
  ```